app/assets/stylesheets/catarse_bootstrap/_main.scss

```diff
@@ -5510,3 +5530,21 @@ html.w-mod-js *[data-ix="display-0-on-load"] {
   }
 }
 
+@font-face {
```

Comment:
Ok! I will do this next time
On 29 Jun 2016 03:52, "Vinícius Andrade" <notifications@github.com> wrote:

> In app/assets/stylesheets/catarse_bootstrap/_main.scss
> <https://github.com/catarse/catarse/pull/2211#discussion_r68807701>:
>
> > @@ -5510,3 +5530,21 @@ html.w-mod-js *[data-ix="display-0-on-load"] {
> >    }
> >  }
> >
> > +@font-face {
>
> Fixed on master.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/catarse/catarse/pull/2211/files/b422fddb0449ab4de7edd427277e9b78276e6871#r68807701>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIROD7ccnzMyR3JU1hfqsfnysEdrwOysks5qQV9cgaJpZM4I_5-H>
> .
>